torchgeo/datasets/geo.py

```diff
@@ -418,10 +435,10 @@ def __getitem__(self, query: BoundingBox) -> Dict[str, Any]:
                             filename = filename[:start] + "*" + filename[end:]
                     filepath = glob.glob(os.path.join(directory, filename))[0]
                     band_filepaths.append(filepath)
-                data_list.append(self._merge_files(band_filepaths, query))
+                data_list.append(self._merge_files(band_filepaths, query, [1]))
```

Comment:
Why use `[1]` instead of the default?

Reply:
Hmm must be a remainder from when I was debugging this. Removed